### PR TITLE
Add de.leopoldluley.Clapgrep

### PR DESCRIPTION
**Pull request by luleyleo**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  
Clapgrep is an app to search through file contents in a directory, including document files like PDF and Office documents. So it's similar to ripgrep-all, but as a GUI application (and not as complete yet).

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: de.leopoldluley.Clapgrep.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      "name": "clapgrep",
+      "buildsystem": "simple",
+      "build-options": {
+        "env": {
+          "CARGO_HOME": "/run/build/clapgrep/cargo"
+        }
+      },
+      "build-commands": [
+        "make --file=build-aux/Makefile"
+      ],
```
Comment (bbhtt) [inline de.leopoldluley.Clapgrep.json:57]: drop `buildsystem: simple` use:

```
        "no-autogen": true,
        "make-args": [
           "--file=build-aux/Makefile"
        ],
```

**Comment by luleyleo** [comment]
Do I still have to do something? Or is it good like this?

**Comment by bbhtt** [comment]
bot, build de.leopoldluley.Clapgrep

**Comment by bbhtt** [comment]
bot, build de.leopoldluley.Clapgrep

**Comment by bbhtt** [comment]
No, looks like it went out of our radar.

**Pull request closed without merging**